esp-hal-common/ld/esp32h2/memory.x

```diff
@@ -16,12 +16,11 @@ MEMORY
 
 
     /* 320K of on soc RAM, 16K reserved for cache */
-    ICACHE : ORIGIN = 0x40800000,  LENGTH = 16K
     /* Instruction and Data RAM 
     0x4086E610 = 2nd stage bootloader iram_loader_seg start address
     see https://github.com/espressif/esp-idf/blob/03414a15508036c8fc0f51642aed7a264e9527df/components/esp_system/ld/esp32h2/memory.ld.in#L26
     */
-    RAM : ORIGIN = 0x40800000 + 16K, LENGTH = 0x3EFD0 - 16K
```

Comment:
If cache isn't here, where is it located? Is it a completely separate region of memory? Maybe we could document this so we don't get confused in the future.

Reply:
I would love to know where the cache is - the TRM just says there is a cache and a brief look into esp-idf only revealed there is no cache at start of RAM. We should howver find an answer to that

Reply:
Have confirmed with the chip designers that the cache data array is not shared with RAM, starting from C6.